logsearch/handlers.py

```diff
@@ -34,6 +35,7 @@ def __str__(self):
             ]
         )
         t.align = "l"
+        t.max_table_width = os.get_terminal_size().columns - 1
```

Comment:
As the [doc](https://docs.python.org/3/library/os.html#os.get_terminal_size) suggest we should use [shutil.get_terminal_size](https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size) instead as that handles couple of edge cases better.

Reply:
Done.